src/containers/topbutton/Top.css

```diff
@@ -0,0 +1,20 @@
+#topButton {
+  /*display: none; /* Hidden by default */
+  /*display: none;*/
```

Comment:
Please uncomment these. Go to top button is visible when the site loads. 
It should not be visible